R/01-estimateMap.R

```diff
@@ -2600,6 +2600,16 @@ estimateMap3DKernel <- function(data,
       kde(cbind(data3$Longitude, data3$Latitude, data3$Date2), H = H)}), silent = TRUE)
   }
   if(clusterMethod == "kmeans"){
+
```

Comment:
Thanks for adding comments and links to the code! This will help in future to understand why clustering was implemented in this way.
Could you add `----` add at the end of (sub)titles? This helps navigating if one uses the doc outline in RStudio:
![image](https://user-images.githubusercontent.com/16759098/233578908-96d6bdc9-0312-421c-8c10-9393e24856ac.png)

![image](https://user-images.githubusercontent.com/16759098/233578803-0163ecdf-866b-4c67-9bd0-3cc21a4123b2.png)